src/java/edu/slu/tpen/transfer/JsonLDExporter.java

```diff
@@ -58,28 +59,36 @@ public JsonLDExporter(Project proj, User u, String profile) throws SQLException,
 {   
    
        Folio[] folios = proj.getFolios();
+
        int projID = proj.getProjectID();
+       String projName = getRbTok("SERVERURL") + "manifest/"+projID;
        try{
-       if (profile.contains("v3")){       
-               manifestData = new LinkedHashMap<>();
-               String projName = getRbTok("SERVERURL") + "manifest/"+projID;
-               
-               manifestData.put("@context", "http://iiif.io/api/presentation/3/context.json");
-               // Fix value of context
-               manifestData.put("id", projName + "/manifest.json");
-               manifestData.put("type", "Manifest");
-               // Not preferred value
-               //Remember that this is a Metadata title, not project name...
-               manifestData.put("label", JsonHelper.buildNoneLanguageMap(proj.getProjectName()));
-               manifestData.put("metadata", getMetadataAsJSON(projID, profile));
-		List<Map<String, Object>> canvasList = new ArrayList<>();
-		int index = 0;
-		for (Folio f : folios) {
-		    System.out.println(f.folioNumber);
-		    index++;
-		    canvasList.add(JsonHelper.buildPage(proj.getProjectID(), projName, f, u, "v3"));
+       if (profile.contains("v3")){
+           List<Map<String, Object>> pageList = new ArrayList<>();
+           for (Folio f : folios) {
+                pageList.add(JsonHelper.buildPage(proj.getProjectID(), projName, f, u,"A"));
+            }
+            //System.out.println("Put all canvas together");
+          
+            manifestData.put("items", pageList);
```

Comment:
Here manifestData is called before being initialized, which causes 500 internal server error when requesting the manifest, so line `manifestData = new LinkedHashMap<>();` should be above this one